### ISSUE TITLE
Propose to change the subsection title in docs to api name

### ISSUE THREAD

**Issue by Ir1d** [williamFalcon/pytorch-lightning#131]
For example in here： https://williamfalcon.github.io/pytorch-lightning/Trainer/Training%20Loop
Personally I think it's clearer if we use the name of the key instead of the usage of the key.

which is, `accumulate_grad_batches` instead of `Accumulated gradients`

**Comment by williamFalcon** [comment]
I hear you, i think it's a good suggestion but the readme is broken down into concrete functionality someone might want to add. one of those functionalities may require multiple flags. if we rename to only the flags, it won't be clear what the framework can do.

the readme is optimized for: How do I accumulate gradients in lightning?
vs: ooh, what does this flag do

with this context in mind, i'd rather keep it how it is right now.

But I agree, if it becomes a big issue later, we can rename
